src/frame/headers.rs

```diff
@@ -343,6 +342,17 @@ impl<T> From<Headers> for Frame<T> {
     }
 }
 
+impl fmt::Debug for Headers {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        f.debug_struct("Headers")
+            .field("stream_id", &self.stream_id)
+            .field("stream_dep", &self.stream_dep)
+            .field("flags", &self.flags)
+            // `fields` and `pseudo` purposefully not included
```

Comment:
Depends.

Would a path or query ever contain a secret? Probably. So Psuedos should be excluded.
Would a header ever be sent that isn't marked sensitive that could include a secret? Like `Authorization`, `Proxy-Authorization`, or any custom header?